.github/workflows/release.yaml

```diff
@@ -66,6 +68,7 @@ jobs:
           HUSKY: 0
           GITHUB_TOKEN: ${{ secrets.LIT_ROBOT_AUTOMATION_PAT }}
           NPM_TOKEN: ${{ secrets.LIT_ROBOT_NPM_PUBLISH_ACCESS_TOKEN }}
+          NPM_CONFIG_PROVENANCE: true
```

Comment:
This is the recommended way of generating provenance when not using `npm publish` directly: https://docs.npmjs.com/generating-provenance-statements#using-third-party-package-publishing-tools